samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -52,6 +52,8 @@ CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE=y
 CONFIG_BT_MESH_DK_PROV=y
 CONFIG_BT_MESH_NLC_PERF_CONF=y
 CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL=y
+CONFIG_BT_MESH_V1d1=y
+CONFIG_BT_MESH_COMP_PAGE_2=y
```

Comment:
I would suggest to move this inside and use `BT_MESH_NLC_PERF_CONF` to derive the `y` value. That way customers have one less item to configure in "prj.conf". Something like this:

nrf\subsys\bluetooth\mesh\Kconfig
```
config BT_MESH_NLC_PERF_CONF
   ...
   select BT_MESH_COMP_PAGE_2
   ...
```

Reply:
I was thinking the same, the reason I did not do it was to show that we now enable Mesh-v1.1. Will move the settings.